libraries/AP_HAL/I2CDevice.h

```diff
@@ -63,6 +63,11 @@ class I2CDevice : public Device {
 
     /* See Device::get_fd() */
     virtual int get_fd() override = 0;
+
+    void ignore_errors(bool b) { _ignore_errors = b; }
+
+protected:
+    bool _ignore_errors;
```

Comment:
if this interface is changed, it needs to be in a separate commit.

In this particular case it's a bad change since we do nothing with that info.